### ISSUE TITLE
Automatic install fails with current HEAD

### ISSUE THREAD

**Issue by TheGreenkey** [1337-server/automatic-ripping-machine#169]
- [x] I have checked the wiki
- [x] I have searched for similar issues

**Describe the bug**
Automatic install under fresh ubuntu 20.04 install with third party drivers included doesn't work on the latest v2_devel. It's not possible to open the `/setup` page to create the db and necessary directories because the service is constantly exiting on startup.

The install worked fine yesterday in a VM under the same conditions. Modifying the install script to revert the repository to commit `1f97a32412ec0831d208483f80ded1342856271e` with `git reset --hard 1f97a32412ec0831d208483f80ded1342856271e` in `function clone_arm()` after the `cd arm` produces a working installation.

**To Reproduce**
1. Fresh ubuntu 20.04 install
2. Install third party drivers (manually now via `apt-get install ubuntu-restricted-extras`
3. Install lsscsi
4. Download and run ubuntu-20.04-install.sh from https://raw.githubusercontent.com/automatic-ripping-machine/automatic-ripping-machine/v2_devel/scripts/ubuntu-20.04-install.sh

### Environment
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"
```
[ARM Release Version or if cloning from git branch/commit (run "git branch" and "git log -1" to get this)] 
Not sure what version exactly I'm getting if I install via the install script. But I didn't specify `-d` in the script. Let's just say the version that is distributed if I ran the script at Sun 08 May 2022 12:34:48 PM UTC

### Log file
Error constantly getting logged in the systemd service/journal:
```
May 08 12:04:41 arm python3[6969]: Traceback (most recent call last):
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3256, in _wrap_pool_connect
May 08 12:04:41 arm python3[6969]:     return fn()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 310, in connect
May 08 12:04:41 arm python3[6969]:     return _ConnectionFairy._checkout(self)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
May 08 12:04:41 arm python3[6969]:     fairy = _ConnectionRecord.checkout(pool)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
May 08 12:04:41 arm python3[6969]:     rec = pool._do_get()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
May 08 12:04:41 arm python3[6969]:     return self._create_connection()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
May 08 12:04:41 arm python3[6969]:     return _ConnectionRecord(self)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
May 08 12:04:41 arm python3[6969]:     self.__connect()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
May 08 12:04:41 arm python3[6969]:     pool.logger.debug("Error on connect(): %s", e)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
May 08 12:04:41 arm python3[6969]:     compat.raise_(
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
May 08 12:04:41 arm python3[6969]:     raise exception
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
May 08 12:04:41 arm python3[6969]:     self.dbapi_connection = connection = pool._invoke_creator(self)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
May 08 12:04:41 arm python3[6969]:     return dialect.connect(*cargs, **cparams)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 597, in connect
May 08 12:04:41 arm python3[6969]:     return self.dbapi.connect(*cargs, **cparams)
May 08 12:04:41 arm python3[6969]: sqlite3.OperationalError: unable to open database file
May 08 12:04:41 arm python3[6969]: The above exception was the direct cause of the following exception:
May 08 12:04:41 arm python3[6969]: Traceback (most recent call last):
May 08 12:04:41 arm python3[6969]:   File "/opt/arm/arm/runui.py", line 8, in <module>
May 08 12:04:41 arm python3[6969]:     import arm.ui.routes  # noqa E402
May 08 12:04:41 arm python3[6969]:   File "/opt/arm/arm/../arm/ui/routes.py", line 29, in <module>
May 08 12:04:41 arm python3[6969]:     armui_cfg = models.UISettings.query.filter_by().first()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 2810, in first
May 08 12:04:41 arm python3[6969]:     return self.limit(1)._iter().first()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 2894, in _iter
May 08 12:04:41 arm python3[6969]:     result = self.session.execute(
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 1691, in execute
May 08 12:04:41 arm python3[6969]:     conn = self._connection_for_bind(bind)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 1532, in _connection_for_bind
May 08 12:04:41 arm python3[6969]:     return self._transaction._connection_for_bind(
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 747, in _connection_for_bind
May 08 12:04:41 arm python3[6969]:     conn = bind.connect()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3210, in connect
May 08 12:04:41 arm python3[6969]:     return self._connection_cls(self, close_with_result=close_with_result)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
May 08 12:04:41 arm python3[6969]:     else engine.raw_connection()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3289, in raw_connection
May 08 12:04:41 arm python3[6969]:     return self._wrap_pool_connect(self.pool.connect, _connection)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3259, in _wrap_pool_connect
May 08 12:04:41 arm python3[6969]:     Connection._handle_dbapi_exception_noconnection(
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 2106, in _handle_dbapi_exception_noconnection
May 08 12:04:41 arm python3[6969]:     util.raise_(
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
May 08 12:04:41 arm python3[6969]:     raise exception
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3256, in _wrap_pool_connect
May 08 12:04:41 arm python3[6969]:     return fn()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 310, in connect
May 08 12:04:41 arm python3[6969]:     return _ConnectionFairy._checkout(self)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
May 08 12:04:41 arm python3[6969]:     fairy = _ConnectionRecord.checkout(pool)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
May 08 12:04:41 arm python3[6969]:     rec = pool._do_get()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
May 08 12:04:41 arm python3[6969]:     return self._create_connection()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
May 08 12:04:41 arm python3[6969]:     return _ConnectionRecord(self)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
May 08 12:04:41 arm python3[6969]:     self.__connect()
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
May 08 12:04:41 arm python3[6969]:     pool.logger.debug("Error on connect(): %s", e)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
May 08 12:04:41 arm python3[6969]:     compat.raise_(
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
May 08 12:04:41 arm python3[6969]:     raise exception
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
May 08 12:04:41 arm python3[6969]:     self.dbapi_connection = connection = pool._invoke_creator(self)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
May 08 12:04:41 arm python3[6969]:     return dialect.connect(*cargs, **cparams)
May 08 12:04:41 arm python3[6969]:   File "/home/arm/.local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 597, in connect
May 08 12:04:41 arm python3[6969]:     return self.dbapi.connect(*cargs, **cparams)
May 08 12:04:41 arm python3[6969]: sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
May 08 12:04:41 arm python3[6969]: (Background on this error at: https://sqlalche.me/e/14/e3q8)
```

**Comment by 1337-server** [comment]
It was the ARM UI settings that triggers this. I've added a hotfix that should fix it until I have a better solution.

**Comment by 1337-server** [comment]
I've also fixed the git clone link, as the main repo shouldn't be linking to here.

**Comment by 1337-server** [comment]
Should be fixed with a01cc462e96a5f287e564ed89c6e0d1e0dd8d86e
